frigate/config.py

```diff
@@ -513,12 +514,22 @@ class CameraMqttConfig(FrigateBaseModel):
 
 
 class RtmpConfig(FrigateBaseModel):
-    enabled: bool = Field(default=True, title="RTMP restreaming enabled.")
+    enabled: bool = Field(default=False, title="RTMP restreaming enabled.")
+
+
+class RestreamConfig(FrigateBaseModel):
+    enabled: bool = Field(default=True, title="Restreaming enabled.")
+
+
+class CameraLiveSourceEnum(str, Enum):
+    jsmpeg = "jsmpeg"
+    restream = "restream"
```

Comment:
Instead of just being restream, could maybe make these have more options like `webRTC`, `mp4`, etc. 

Difficulty is that webRTC requires frigate to be run as host, so that may be confusing 🤔